ens/src/lib.rs

```diff
@@ -290,27 +291,29 @@ async fn setup(
     let call = resolver.multicall(calls)?;
     ethereum::transaction(call).await?;
 
-    let spinner = term::spinner("Updating local identity...");
-    match person::set_ens_payload(
-        person::Ens {
-            name: name.to_owned(),
-        },
-        storage,
-    ) {
-        Ok(doc) => {
-            spinner.finish();
-            term::blob(serde_json::to_string(&doc.payload())?);
-        }
-        Err(err) => {
-            spinner.failed();
-            return Err(err);
+    if chain_id == 1 {
```

Comment:
Makes sense but this function doesn't do anything else, so we should return an error instead.

Reply:
Also check the `Chain` type un the ethereum module to convert the uint.

Reply:
Hm... it updates local identity. I don't want to pollute it when working with test chains.
![Screenshot from 2022-05-02 17-32-01](https://user-images.githubusercontent.com/14107758/166237965-c6591a06-223b-4a88-8edf-080863d021f4.png)

Reply:
Yes I agree, but maybe include a warning then `Warning: Skipping local ENS setup`.

Reply:
Done